packages/generator/generator.js

```diff
@@ -366,9 +376,12 @@ async function generateFontPackage(webfont) {
         version: PackageVersion,
         description: `Use the ${webfont.family} font family from Google Fonts in your Expo app`,
         main: 'index.js',
-        repository: 'https://github.com/expo/google-fonts.git',
-        homepage: `https://github.com/expo/google-fonts/tree/master/font-packages/${packageName}#readme`,
-        author: 'Expo Team <team@expo.io>',
+        repository: {
```

Comment:
@kadikraman  These change is also something not easy to anticipate, I think npm change there parsing behavior, if you just put `repository` like before, pushing on npm gives an warning